friends/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state= {
+      friends: []
+    };
+  }
+  componentDidMount() {
```

Comment:
In the instructions, item 6: `Inside your React application, create a component to display the list of friends coming from the server.` This CDM and the JSX for rendering the friends should have been in a separate component. Perhaps one called FriendsList.